app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -335,10 +335,10 @@ export default {
       if (this.isAWebWidgetInbox) {
         return [
           ...visibleToAllChannelTabs,
-          // {
-          //   key: 'campaign',
-          //   name: this.$t('INBOX_MGMT.TABS.CAMPAIGN'),
-          // },
+          {
+            key: 'campaign',
+            name: this.$t('INBOX_MGMT.TABS.CAMPAIGN'),
+          },
```

Comment:
Let us remove this and add this as a separate PR once the widget side is merged.